release.py

```diff
@@ -306,7 +343,7 @@ def select_gpg_key():
 
 cmd("Building artifacts", "gradle", cwd=kafka_dir, env=jdk7_env)
 cmd("Building artifacts", "./gradlew clean releaseTarGzAll aggregatedJavadoc", cwd=kafka_dir, env=jdk7_env)
-# This should be removed with KAFKA-4421
+# This should be removed when Java7 is dropped (cf. KAFKA-4421)
```

Comment:
new

Reply:
this is not a comment: https://issues.apache.org/jira/browse/KAFKA-4421 is already merged in trunk, so we should modify this place if wanting to cherry pick to newer branches.

Reply:
👍  was going to make the exact same comment

Reply:
Yes. This PR is for 0.11.0 and should similar for 1.0 and 1.1 -- for 2.0 and `trunk` we need to update the script accordingly.

I'll merge this PR as well as #5347 and #5328 and move on to forward port this to other branches.